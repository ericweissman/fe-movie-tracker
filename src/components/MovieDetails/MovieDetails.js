import React from 'react'

const MovieDetails = ({ title, poster_path, release_date, vote_average, overview }) => {
  const movieImageUrl = "https://image.tmdb.org/t/p/w500"
  return(
    <div className='movie-details'>
      <div className='movie-details-card'>
        <div className='title-area'>
          <h3>{title}</h3>
          <img src={movieImageUrl + poster_path} alt="movie poster" />
        </div>
        <div className='details'>
          <p className='overview'> <span>Synopsis:</span> {overview}</p>
          <p><span>Released:</span> {release_date}</p>
          <p><span>Average Score:</span> {vote_average}</p>
        </div>
      </div>
    </div>

  )
}

export default MovieDetails;
