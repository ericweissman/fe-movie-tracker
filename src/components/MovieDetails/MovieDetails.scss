.movie-details {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 20px 0;
}

.movie-details-card {
  display: grid;
  grid-template-columns: 350px 1fr;
  font-family: 'Roboto', sans-serif;
  border: 3px solid;
  border-radius: 10px;
  width: 1000px;
  span {
    font-weight: 600;
  }
  h3 {
    font-size: 30px;
    margin: 15px 0px;
  }

  .title-area

  .overview {
    width: 500px;
    text-align: left;
  }
  p {
    text-align: left;
    font-size: 20px;
    padding-right: 100px;
    line-height: 30px;
  }

  img {
    height: 350px;
    width: 250px;
    margin-bottom: 15px;
    border-radius: 5px;
  }
}