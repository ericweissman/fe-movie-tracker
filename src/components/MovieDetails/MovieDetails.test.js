import { shallow } from 'enzyme';
import MovieDetails from './MovieDetails'
import React from 'react'

describe('MovieDetails', () => {
  let wrapper

  it('should match the correct snapshot', () => {
    wrapper = shallow(<MovieDetails />)
    expect(wrapper).toMatchSnapshot();
  })
})