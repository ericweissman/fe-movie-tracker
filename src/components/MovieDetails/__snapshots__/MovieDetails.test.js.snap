// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails should match the correct snapshot 1`] = `
<div
  className="movie-details"
>
  <div
    className="movie-details-card"
  >
    <div
      className="title-area"
    >
      <h3 />
      <img
        alt="movie poster"
        src="https://image.tmdb.org/t/p/w500undefined"
      />
    </div>
    <div
      className="details"
    >
      <p
        className="overview"
      >
         
        <span>
          Synopsis:
        </span>
         
      </p>
      <p>
        <span>
          Released:
        </span>
         
      </p>
      <p>
        <span>
          Average Score:
        </span>
         
      </p>
    </div>
  </div>
</div>
`;
